src/client/entity-editor/common/entity.js

```diff
@@ -19,49 +19,52 @@
 
 // @flow
 
-import Icon from 'react-fontawesome';
+// import Icon from 'react-fontawesome';
 import React from 'react';
 import {genEntityIconHTMLElement} from '../../helpers/entity';
 
 
 type EntityProps = {
 	disambiguation?: ?string,
-	link?: string | false,
 	text: string,
 	type: string,
 	unnamedText?: string
 };
 
 function Entity(
-	{disambiguation, link, text, type, unnamedText}: EntityProps
+	{disambiguation, id, text, type, unnamedText}: EntityProps
 ) {
-	const nameComponent = text || <i>{unnamedText}</i>;
+	const url = type === 'Area' ?
+		`//musicbrainz.org/area/${id}` :
+		`/${type.toLowerCase()}/${id}`;
+	// const nameComponent = text || <i>{unnamedText}</i>;
 	const contents = (
 		<span>
 			{
 				type && genEntityIconHTMLElement(type)
 			}
 			{' '}
-			{nameComponent}
-			{' '}
+			<a href={url} rel="noopener noreferrer" target="_blank">
+				{text} &nbsp; ( {type} )
+			</a>
+
 			{
 				disambiguation &&
 				<span className="disambig"><small>({disambiguation})</small></span>
 			}
 		</span>
 	);
 
-	if (link) {
-		return <a href={link}>{contents}</a>;
-	}
+	// if (link) {
```

Comment:
You forgot to clean up after yourself :p

Reply:
Actually, I thought may it have some use in future that y I don't delete it. I will clean it in next commit.